test/deploy/validate-configs.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# Script to validate the packaged module
+
+TEST_NAME="Deployment testing"
+mkdir logs
+
+echo "$TEST_NAME - Building and installing package"
+npm pack --loglevel warn
+npm install -g solid-community-server-*.tgz --loglevel warn
+rm solid-community-server-*.tgz
+
+run_server_with_config () {
+  if [[ $# -ne 2 ]]; then
+    echo "Config arguments not set"
+    return 1
+  elif [ ! -f $1 ]; then
+    echo "Config file does not exist, check the path."
+    return 1
+  fi
+  CONFIG_PATH=$1
+  CONFIG_NAME=$2
+
+  CSS_OPTS="-p 8888 -l warn -s http://localhost:4000/sparql"
+  CURL_STATEMENT='http://localhost:8888'
+  # HTTPS config needs a base-url override + keys and certs
+  if [[ $CONFIG_NAME =~ "https" ]]; then
+    sed -i -E "s/(\W+\"options_key\".*\").+(\".*)/\1server.key\2/" $CONFIG_PATH
+    sed -i -E "s/(\W+\"options_cert\".*\").+(\".*)/\1server.cert\2/" $CONFIG_PATH
+    CSS_OPTS="$CSS_OPTS -b https://localhost:8888 --httpsKey server.key --httpsCert server.cert"
+    CURL_STATEMENT='-k https://localhost:8888'
+  fi
```

Comment:
I would have 2 separate if-statements for the 2 different HTTPS configs so only one of them gets the CLI params, and only one of them gets the changed config. You can also changes the paths in `example-https-file.json` should that make testing easier.